src/canvas.tsx

```diff
@@ -0,0 +1,158 @@
+/**
+ * @license qrcode.react
+ * Copyright (c) Paul O'Shannessy
+ * SPDX-License-Identifier: ISC
+ */
+
+import React, {useRef, useEffect} from 'react';
```

Comment:
```suggestion
import * as React from 'react';
import {useRef, useEffect} from 'react';
```

or

```suggestion
import * as React from 'react';
```
if yu want to use `React.useRef` / `React.useEffect`. Default export from `react` might be removed in v20 or a bit later. It’s not tree-shakeable.